docs/application/dotnet/guides/user-interface/nui/overview.md

```diff
@@ -13,31 +13,35 @@ After you have set up the NUI development environment, you can quickly create ri
 
 NUI offers the following main features:
 
-- [Layer](layer.md)
+- [Animation](animation.md)
 
-  Provides a mechanism for overlaying groups of views on top of each other.
+  Allows objects to move around and change their properties for a specified duration.
 
-- [Window](window.md)
+- [Components](ui-components.md)
 
-  Creates a default window for building the main user interface of the application and provides various events such as key event and touch event to the application.
+  Interactive components for layout and user interface, such as buttons, table view, text controls, image view, flex container, slider, and video view.
 
-- [Layouts](layouts.md)
+- [Layer](layer.md)
 
-  Provide advanced reusable laying out capabilities.
+  Provides a mechanism for overlaying groups of views on top of each other.
 
-- [UI Components](ui-components.md)
+- [Layouts](layouts.md)
 
-  Interactive components for layout and user interface, such as buttons, table view, text controls, image view, flex container, slider, and video view.
+  Provides advanced reusable laying out capabilities.
 
-- [Animation](animation.md)
+- [Palette](palette.md)
```

Comment:
Although I can see "Palette" on the left navigation panel in the overview page, once I click the hyperlink, the file does not show up in the left navigation panel anymore.

**Overview screenshot:**
![image](https://user-images.githubusercontent.com/94053723/146901951-dda8965e-472d-45f3-9c7e-e63db2959981.png)

**Screenshot after clicking on hyperlink:**
![image](https://user-images.githubusercontent.com/94053723/146900977-84949ba9-059c-4851-9227-bd204731efab.png)